packages/components/src/toggle-group-control/stories/index.tsx

```diff
@@ -81,6 +81,7 @@ Default.args = {
 		{ value: 'right', label: 'Right' },
 		{ value: 'justify', label: 'Justify' },
 	].map( mapPropsToOptionComponent ),
+	isBlock: true,
```

Comment:
Added this so it matches our "recommended default". (Note that `isBlock` is similarly enabled in our README code snippets.)